fw-child/template/query/variable-item.php

```diff
@@ -59,35 +67,35 @@
 		</div>
 	</div>
 	
-	<div class="flex-drawer-content d-none">
+	<div class="flex-drawer-content d-none variable-details">
 		<div class="row">
-			<div class="col-5-of-12 offset-1-of-12 mb-5 pb-5 border-bottom border-gray-500">
+			<div class="col col-lg-6 col-14 offset-1 variable-details__title">
 				<h2 class="mb-0 font-family-serif text-secondary"><?php echo get_the_title ( $item['id'] ); ?></h2>
 			</div>
-			
-			<div class="col-5-of-12 d-flex align-items-center mb-5 pb-5 border-bottom border-gray-500">
-				<div class="offset-1-of-5 d-flex align-items-center">
-					<?php
-					
-						$map_slug = 'map/';
-						$dl_slug = 'download/';
-						
-						if ( $GLOBALS['fw']['current_lang_code'] == 'fr' ) {
-							$map_slug = 'carte/';
-							$dl_slug = 'telechargement/';
-						}
-						
-					?>
-					
-					<a href="<?php echo home_url ( $map_slug ); ?>?var_id=<?php echo $item['id']; ?>" class="btn btn-primary rounded-pill px-4 me-3"><?php _e ( 'View on Map', 'cdc' ); ?></a>
-					
-					<a href="<?php echo home_url ( $dl_slug ); ?>?var_id=<?php echo $item['id']; ?>" class="btn btn-light rounded-pill px-4"><?php _e ( 'Download Data', 'cdc' ); ?></a>
-				</div>
+
+			<div class="col col-lg-8 offset-lg-0 col-14 offset-1 variable-details__links">
+				<?php
+				$map_slug = 'map/';
+				$dl_slug  = 'download/';
+
+				if ( $GLOBALS['fw']['current_lang_code'] == 'fr' ) {
```

Comment:
To be consistent with the rest of this file, we should use the triple `===`, and put `'fr'` on the left:
(I don't know why GitHub shows differences in indentation though, so ignore it...)

 ```suggestion
				if ( 'fr' === $GLOBALS['fw']['current_lang_code'] ) {
```